metadata-ingestion/src/datahub/ingestion/source/state/stateful_ingestion_base.py

```diff
@@ -52,7 +52,7 @@ class StatefulIngestionConfig(ConfigModel):
 
     enabled: bool = Field(
         default=False,
-        description="The type of the ingestion state provider registered with datahub.",
+        description="Default as True if datahub-rest sink is used or if datahub_api is specified, otherwise False",
```

Comment:
```suggestion
        description="Whether or not to enable stateful ingest. Default: True if datahub-rest sink is used or if a `datahub_api` is specified, otherwise False",
```